src/main/java/uk/gov/pay/adminusers/client/ledger/model/LedgerSearchTransactionsResponse.java

```diff
@@ -0,0 +1,34 @@
+package uk.gov.pay.adminusers.client.ledger.model;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.PropertyNamingStrategies;
+import com.fasterxml.jackson.databind.annotation.JsonNaming;
+
+import java.util.List;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+@JsonNaming(PropertyNamingStrategies.SnakeCaseStrategy.class)
+public class LedgerSearchTransactionsResponse {
+
+    @JsonProperty("total")
+    private int total;
+
+    @JsonProperty("count")
+    private int count;
```

Comment:
What's the difference between `total` and `count`?

Reply:
total is the number of results for the search and count will be the page size. I have removed these 2 fields as these are not being used.